app/src/main/java/org/groundplatform/android/ui/map/gms/GoogleMapsFragment.kt

```diff
@@ -241,7 +246,31 @@ class GoogleMapsFragment : SupportMapFragment(), MapFragment {
 
   override fun setFeatures(newFeatures: Set<Feature>) {
     Timber.v("setFeatures() called with ${newFeatures.size} features")
-    featureManager.setFeatures(newFeatures)
+
+    val draft = newFeatures.firstOrNull { it.isDraftLineString() }
```

Comment:
Sorry to keep pushing on this, but this isn't the right place either. This whole package implements the bridge between Maps SDK and our own geometry model and events. "Draft drawing state" has no meaning in this context; it just shows the features it's given, with the style it's given.

Since this behavior is specific to the "Draw area" feature, I would expect it to be implemented in the `DrawAreaTaskViewModel`, not here in the map adapter.